addons/website_event/static/src/js/website_event_send_email.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+
+import { rpc } from "@web/core/network/rpc";
+import { checkEmailValidity } from  "@mail/utils/common/format";
+import publicWidget from '@web/legacy/js/public/public_widget';
+
+publicWidget.registry.websiteEventSendEmail = publicWidget.Widget.extend({
+    selector: '.o_wevent_js_send_email',
+    events: {
+        'click': '_onSendEmailClick',
+    },
+
+    init() {
+        this.notification = this.bindService("notification");
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onSendEmailClick: async function (ev) {
+        const eventId = parseInt(ev.currentTarget.dataset.eventId, 10);
+        const attendeesIds = JSON.parse(ev.currentTarget.dataset.attendeesIds);
+        const ticketsHash = ev.currentTarget.dataset.ticketsHash;
+        const emails = document.querySelector('#o_send_by_email_input').value;
+        const emailInfo = checkEmailValidity(', ', emails)
+
+        if (emails === "" || emailInfo.invalidEmails.length) {
+            this._showError();
+        } else {
+            this._hideError();
+            await rpc(`/event/${ eventId }/my_tickets_by_email`, {
+                'registration_ids': attendeesIds,
+                'tickets_hash': ticketsHash,
+                'emails': emails
+            });
+            this._showInfoMsg();
+        }
+    },
+
+    _showError: function () {
+        const inputElement = document.querySelector('#o_send_by_email_input');
+        inputElement.classList.add('is-invalid');
+        inputElement.parentElement.classList.add('is-invalid');
```

Comment:
I would not mind a line more to be more explicit on what is the parentElement (same below)